test/odoc_print/odoc_print.ml

```diff
@@ -204,9 +256,37 @@ let a_inp =
   let doc = "Input file." in
   Arg.(required & pos 0 (some file) None & info ~doc ~docv:"PATH" [])
 
+let a_short =
+  let doc = "Short output." in
+  Arg.(value & flag & info ~doc [ "short" ])
+
+let a_show_expansions =
+  let doc = "Show expansions in short output" in
+  Arg.(value & flag & info ~doc [ "show-expansions" ])
+
+let a_long_paths =
+  let doc = "Show long paths in short output" in
+  Arg.(value & flag & info ~doc [ "long-paths" ])
+
+let a_show_canonical =
+  let doc = "Show modules canonical reference in short output" in
+  Arg.(value & flag & info ~doc [ "show-canonical" ])
+
+let a_show_include_expansions =
+  let doc = "Show include expansions in short output" in
+  Arg.(value & flag & info ~doc [ "show-include-expansions" ])
```

Comment:
What's the inconvenient in merging this option with `--show-expansions` ? The output is quite short already.